recipe/meta.yaml

```diff
@@ -11,11 +11,12 @@ source:
   sha256: 3771a65699caee0f5b97cedada74a52b9ba4061bd8ad150d0115defb291c4581
 
 build:
-  number: 0
+  number: 1
```

Comment:
Technically don't need to bump the build number since this is just trying to fix the packages that failed to build the first time. Not doing so would eliminate uploading some duplicate packages for linux/mac. Not a huge deal though.

Reply:
Oh okay! I'll wait for this build to finish and in case there are any changes, will put this in along with it so that I don't stop this build. :)